samples/AnomalyDetector/src/Generated/Models/AlignPolicy.Serialization.cs

```diff
@@ -19,14 +19,22 @@ public partial class AlignPolicy : IUtf8JsonSerializable, IJsonModel<AlignPolicy
         void IUtf8JsonSerializable.Write(Utf8JsonWriter writer) => ((IJsonModel<AlignPolicy>)this).Write(writer, ModelSerializationExtensions.WireOptions);
 
         void IJsonModel<AlignPolicy>.Write(Utf8JsonWriter writer, ModelReaderWriterOptions options)
```

Comment:
Remember delete all the configuration i, e.g., test\CadlRanchProjects\type\model\inheritance\enum-discriminator\tspconfig.yaml

Reply:
I did a global research in vscode and there is only one 'use-write-core: true' left.